documentation/modules/post/linux/gather/haserl_read.md

```diff
@@ -0,0 +1,59 @@
+## Vulnerable Application
+
+[Haserl](http://haserl.sourceforge.net/) is an unmaintained tool to use LUA as CGI in web servers.
+On Linux, if haserl is usually suid root, sot that it will attempt to drop its privilege to the uid/gid of the owner of the cgi script,
+similar to suexec in Apache. 
+
+Haserl could have been a thing of the past, but it's used in Alpine Linux'
+[Alpine Configuration
+Framework](https://wiki.alpinelinux.org/wiki/Alpine_Configuration_Framework_Design),
+which is commonly used on this distribution.
+
+This module exploits the fact that calling haserl on a file will make it not only not drop its priviledges,
+but also display the content of the file.
+
+### Prerequisites
+
+1. Install Alpine Linux
+2. Install haserl
+
+## Verification Steps
+
+1. Start msfconsole
+2. Get a shell
+2. Do: `use exploit/linux/gather/haserl_read`
+3. Set `SESSION`
+4. Do: `run` or `exploit`
+5. **Verify** that the file was successfully downloaded
```

Comment:
`2` is duplicated. Using `1` for ever entry is acceptable. Markdown renderers will automatically number the list items appropriately.